data/prefs.glade

```diff
@@ -3827,22 +3827,63 @@ Control-H</property>
 			  <property name="spacing">0</property>
 
 			  <child>
-			    <widget class="GtkLabel" id="label_hooks_description">
+			    <widget class="GtkVBox" id="vbox27">
 			      <property name="visible">True</property>
-			      <property name="label" translatable="yes">Hooks</property>
-			      <property name="use_underline">False</property>
-			      <property name="use_markup">False</property>
-			      <property name="justify">GTK_JUSTIFY_LEFT</property>
-			      <property name="wrap">False</property>
-			      <property name="selectable">False</property>
-			      <property name="xalign">0.5</property>
-			      <property name="yalign">0.5</property>
-			      <property name="xpad">10</property>
-			      <property name="ypad">10</property>
-			      <property name="ellipsize">PANGO_ELLIPSIZE_NONE</property>
-			      <property name="width_chars">-1</property>
-			      <property name="single_line_mode">False</property>
-			      <property name="angle">0</property>
+			      <property name="homogeneous">False</property>
+			      <property name="spacing">0</property>
+
+			      <child>
+				<widget class="GtkLabel" id="label4827">
+				  <property name="visible">True</property>
+				  <property name="label" translatable="yes">&lt;b&gt;Hooks&lt;/b&gt;
+&lt;small&gt;In this dialog your can attach your own scripts to various guake events.
+For details, see &lt;/small&gt;</property>
+				  <property name="use_underline">False</property>
+				  <property name="use_markup">True</property>
+				  <property name="justify">GTK_JUSTIFY_CENTER</property>
+				  <property name="wrap">True</property>
+				  <property name="selectable">False</property>
+				  <property name="xalign">0.5</property>
+				  <property name="yalign">0.5</property>
+				  <property name="xpad">10</property>
+				  <property name="ypad">0</property>
+				  <property name="ellipsize">PANGO_ELLIPSIZE_START</property>
+				  <property name="width_chars">-1</property>
+				  <property name="single_line_mode">False</property>
+				  <property name="angle">0</property>
+				</widget>
+				<packing>
+				  <property name="padding">0</property>
+				  <property name="expand">False</property>
+				  <property name="fill">False</property>
+				</packing>
+			      </child>
+
+			      <child>
+				<widget class="GtkLabel" id="label_hooks_link_to_github">
+				  <property name="visible">True</property>
+				  <property name="can_focus">True</property>
+				  <property name="label" translatable="yes">&lt;small&gt;&lt;b&gt;https://github.com/Guake/guake/issues/793&lt;/b&gt;&lt;/small&gt;</property>
```

Comment:
I must admit I still dont get the point of this change. Reading all this PR is too complex.
I advise:
- label said something like "In this dialog your can attach your own scripts to various guake events, for more information see tooltip of this label"

and you add a more complex documentation in the label's tooltip.